utils/git-sync-deps

```diff
@@ -266,15 +306,14 @@ def multithread(function, list_of_arg_lists):
 def main(argv):
   deps_file_path = os.environ.get('GIT_SYNC_DEPS_PATH', DEFAULT_DEPS_PATH)
   verbose = not bool(os.environ.get('GIT_SYNC_DEPS_QUIET', False))
+  treeless = bool("--treeless" in argv)
+  argv = [x for x in argv if x != "--treeless"]
```

Comment:
Could probably simplify this with `import argparse`, but maybe if the script keeps getting flags added.

Reply:
Fixed with https://github.com/KhronosGroup/SPIRV-Tools/pull/5038